src/base/bittorrent/abstractfilestorage.cpp

```diff
@@ -81,7 +84,7 @@ void BitTorrent::AbstractFileStorage::renameFolder(const Path &oldFolderPath, co
         if (path.hasAncestor(oldFolderPath))
             renamingFileIndexes.append(i);
         else if (path.hasAncestor(newFolderPath))
-            throw RuntimeError(tr("The folder already exists: '%1'.").arg(newFolderPath.toString()));
+            throw RuntimeError(tr("The folder already exists in this torrent: '%1'.").arg(newFolderPath.toString()));
```

Comment:
This message is more confusing. There is no indication of what torrent it is referring to.

Reply:
```suggestion
            throw RuntimeError(tr("The folder path is the same: '%1'.").arg(newFolderPath.toString()));
```